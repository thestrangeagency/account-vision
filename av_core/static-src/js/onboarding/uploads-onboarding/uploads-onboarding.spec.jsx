import React from 'react';
import { shallow } from 'enzyme';

import UploadsOnboarding from './uploads-onboarding';

test('it renders', () => {
  const wrapper = shallow(<UploadsOnboarding nextPage="" returnYear="" />);
  expect(wrapper).toMatchSnapshot();
});
