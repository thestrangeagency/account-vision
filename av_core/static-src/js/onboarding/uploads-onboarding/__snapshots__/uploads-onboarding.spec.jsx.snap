// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="custom-onboarding"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <p>
        The list provided below are some examples of forms that you can upload. Feel free to upload any additional forms you feel our CPAs might need to see. If you have a copy of your tax return from the previous year, please upload that as well.
      </p>
      <ul>
        <li>
          W-2
        </li>
        <li>
          1099-INT: Interest Income
        </li>
        <li>
          1099-DIV: Dividends and Distributions
        </li>
        <li>
          1098, Mortgage Interest Statement,
        </li>
        <li>
          1098-E, Student Loan Interest Statement
        </li>
        <li>
          1099-B: Proceeds from Broker and Barter Exchange Transactions
        </li>
        <li>
          1099-MISC: Miscellaneous Income
        </li>
        <li>
          State Drivers License
        </li>
      </ul>
      <UploadManager
        frozen={false}
        target={null}
        year=""
      />
    </div>
  </div>
  <div
    className="row justify-content-center"
  >
    <div
      className="col col-6"
    >
      <button
        className="btn btn-primary w-100"
        onClick={[Function]}
        type="button"
      >
        Next
      </button>
    </div>
  </div>
</div>
`;
